Src/Private/Get-AbrFgtForticare.ps1

```diff
@@ -0,0 +1,168 @@
+
+function Get-AbrFgtForticare {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns FortiCare settings.
+    .DESCRIPTION
+        Documents the configuration of Fortinet Fortigate in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.1.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.Fortigate
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PscriboMessage "Discovering Forticare settings information from $System."
+    }
+
+    process {
+
+        $Forticare = (Get-FGTMonitorLicenseStatus).forticare
+
+        Section -Style Heading2 'FortiCare' {
+            Paragraph "The following section details Forticare settings configured on Fortigate."
+            BlankLine
+
+            $OutObj = @()
+
+            $Serial = $DefaultFGTConnection.serial
+
+            $OutObj = [pscustomobject]@{
+                "Model"   = $Model
+                "Serial"  = $Serial
+                "Status"  = $Forticare.status
+                "Account" = $Forticare.account.ToLower()
+                "Company" = $Forticare.company
+            }
+
+            $TableParams = @{
+                Name = "FortiCare"
+                List = $true
```

Comment:
All tables must have set column widths. Table names should include the system name or object which they relate to. Please also include the `ShowTableCaptions` option for all tables.

```
$TableParams = @{
    Name = "Forticare - $($System)"
    List = $true
    ColumnWidths = 50, 50
}
if ($Report.ShowTableCaptions) {
    $TableParams['Caption'] = "- $($TableParams.Name)"
}
$OutObj | Table @TableParams
```